contributed_definitions/NXregion.nxdl.xml

```diff
@@ -0,0 +1,193 @@
+					block  = [16]
+					downsampled:NXdata/
+						@signal = "maximum"
+						@auxiliary_signals = "copy"
+						maximum[128,128,20]
+						copy[128,128,320]
+
+		the ``copy`` dataset selects 20 16-channel blocks that start 32 channels apart,
+		the ``maximum`` dataset will show maximum values in each 16-channel block
+		in every spectra.
+	</doc>
+	<attribute name="type" optional="false">
+		<doc>
+			This is ``rectangular`` to describe the region as a hyper-rectangle in
+			the index space of its parent group's data field
+		</doc>
+		<enumeration>
+			<item value="rectangular" />
+		</enumeration>
+	</attribute>
+	<field name="parent">
+		<doc>
+			The name of data field in the parent group or the path of a data field relative
+			to the parent group (so it could be a field in subgroup)
+		</doc>
+	</field>
+	<field name="parent_mask">
+		<doc>
+			The name of an optional mask field in the parent group whose shape matches the
+			region dimensions of the parent data field, i.e. :math:`\boldsymbol{d}`. For example,
+			this could be ``pixel_mask`` of an :ref:`NXdetector`
+		</doc>
+	</field>
+	<field name="start" type="NX_NUMBER" recommended="true">
+		<doc>
+			The starting position for region in detector data field array.
+			This is recommended as it also defines the region rank.
+			If omitted then defined as an array of zeros
+		</doc>
+		<dimensions rank="1">
+			<dim index="1" value="R"/>
+		</dimensions>
+	</field>
+	<field name="count" type="NX_INT" recommended="true">
+		<doc>
+			The number of blocks or items in hyperslab selection.
+			If omitted then defined as an array of dimensions that take into account
+			the other hyperslab selection fields to span the parent data field's shape
+		</doc>
+		<dimensions rank="1">
+			<dim index="1" value="R"/>
+		</dimensions>
+	</field>
+	<field name="stride" type="NX_INT">
+		<doc>
+			An optional field to define striding used to downsample data.
+			If omitted then defined as an array of ones
+		</doc>
+		<dimensions rank="1">
+			<dim index="1" value="R"/>
+		</dimensions>
+	</field>
+	<field name="block" type="NX_INT">
+		<doc>
+			An optional field to define the block size used to copy or downsample data. In the
+			:math:`i`-th dimension, if :math:`\mathbf{block}[i] \lt \mathbf{stride}[i]`
+			then the downsampling blocks have gaps between them; when ``block`` matches ``stride``
+			then the blocks are contiguous; otherwise the blocks overlap.
+			If omitted then defined as an array of ones
+		</doc>
+		<dimensions rank="1">
+			<dim index="1" value="R"/>
+		</dimensions>
+	</field>
+	<field name="scale" type="NX_NUMBER">
+		<doc>
+			An optional field to define a divisor for scaling of reduced data. For example, in a
+			downsampled sum, it can reduce the maximum values to fit in the domain of the result
+			data type. In an image that is downsampled by summing 2x2 blocks, using
+			:math:`\mathrm{scale}=4' allows the result to fit in the same integer type dataset as
```

Comment:
The math doesn't render for me
![image](https://user-images.githubusercontent.com/7264703/174083680-d1df0ec2-3443-4502-9015-05cd3b4a8b74.png)

Reply:
This is the error `=4'`, should end with a backtick.